flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
WHY?

Reply:
With the official release of Zed, aarch64 has been accounted for in the manifest